application/app/helpers/detached_process_helper.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+
+module DetachedProcessHelper
+  def process_status_class(status, type:)
+    if status.idle?
+      OpenStruct.new(
+        button: 'btn-outline-warning',
+        icon: type == :download ? 'bi-arrow-down-circle' : 'bi-arrow-up-circle',
+        text: 'text-warning-emphasis'
+      )
+    else
+      OpenStruct.new(
+        button: 'btn-outline-info',
+        icon: type == :download ? 'bi-arrow-down-circle-fill' : 'bi-arrow-up-circle-fill',
+        text: 'text-info pulse'
```

Comment:
is this `text-info-pulse` instead?

Reply:
No, it is like this. 2 CSS classes. To see them working, schedule a download and see how the icons in the header change as well as the icon in the Uploads and Downloads pages